packages/solana/src/__tests__/index-test.ts

```diff
@@ -48,4 +51,42 @@ describe("TurnkeySigner", () => {
     await signer.addSignature(transferTransaction, turnkeySolAddress);
     expect(transferTransaction.signatures.length).toBe(1);
   });
+
+  test("can sign a message with a Solana account", async () => {
+    if (!process.env.SOLANA_TEST_ORG_API_PRIVATE_KEY) {
+      // This test requires an env var to be set
+      console.warn(
+        "This test is skipped because it cannot run without SOLANA_TEST_ORG_API_PRIVATE_KEY set"
+      );
+      return;
```

Comment:
Can you make this a hard failure? I made that change already on lines 11-14 so you can copy/paste:
```js
throw new Error(
  "This test requires SOLANA_TEST_ORG_API_PRIVATE_KEY to be set"
);
```